frontend/javascripts/oxalis/model/sagas/dataset_saga.ts

```diff
@@ -149,3 +152,31 @@ export function* watchZ1Downsampling(): Saga<void> {
     maybeShowWarning,
   );
 }
+
+export function* ensureSegmentIndexIsLoaded(): Saga<void> {
```

Comment:
I was wondering whether it makes sense to have a big dataset saga (or to start one), but all the other sagas in the module were rather separate so I put mine separately, too.